cpp/src/arrow/c/bridge_test.cc

```diff
@@ -122,7 +125,7 @@ class ReleaseCallback {
  private:
   ARROW_DISALLOW_COPY_AND_ASSIGN(ReleaseCallback);
 
-  bool called_;
+  bool called_{};
```

Comment:
should this be `{ false };` ? (unless I'm forgetting how the default initialization works)

Reply:
It would be more readable like that, yes.